src/redux/store.js

```diff
@@ -1,6 +1,7 @@
 import { configureStore } from '@reduxjs/toolkit';
 import { createLogger } from 'redux-logger';
-import { rocketsSlice } from './rockets/RocketsSlice';
+import missionsReducer from './missions/missionsSlice.js';
```

Comment:
Please, I suggest that you ensure that the name of the file tarry while fetching the function. Please kindly look into it and update it. it should be import missionsReducer from './missions/MissionsSlice.js';

Reply:
@Donmark2k thank you for the feedback file name has been corrected